src/main/scala/Transition.scala

```diff
@@ -0,0 +1,18 @@
+import org.scalajs.dom
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+
+/** @see [[https://github.com/d3/d3-transition]] */
+@JSImport("d3-transition", JSImport.Namespace)
+@js.native
+object d3transition extends js.Object {
+
+  def duration(value: Double): Transition[dom.EventTarget] = js.native
+
+  @js.native
+  trait Transition[Datum] extends js.Object {
```

Comment:
Should this extend `d3v4.d3selection.BaseDom`?

Reply:
It doesn't look like it should. The Typescript transition facade does not support methods like `classed`. See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-transition/v1/index.d.ts#L95 and https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-selection/v1/index.d.ts#L180